babel/index.js

```diff
@@ -127,6 +127,31 @@ module.exports = function({ types: t }) {
           path.replaceWithMultiple(imports);
         }
       },
+      ExportNamedDeclaration(path) {
+        const { source, specifiers } = path.node;
+        if (source.value === 'react-native' && specifiers.length) {
```

Comment:
This breaks when node chooses the `module` path from `package.json`, because the `dist/module.js` doesn't include a `source` for the `export { ... };` statement.

```
Module build failed: TypeError: project/tools/devserver/hmr-entry.js: Cannot read property 'value' of null
    at PluginPass.ExportNamedDeclaration (project/node_modules/react-native-web/babel/index.js:132:19)
```